jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -103,7 +103,8 @@ def buildExclusions = [
   // OSX ---------------------------------------------------
   [ /^osx1010/,                       anyType,     gte(11) ],
   [ /^osx1011/,                       releaseType, lt(11)  ],
-  [ /^osx1011/,                       releaseType, gte(13)  ],
+  [ /^osx1011/,                       releaseType, gte(13) ],
+  [ /^osx1011/,                       anyType,     gte(14) ],
```

Comment:
```suggestion
  [ /^osx1011/,                       testType,     gte(14) ],
```

Reply:
oh didnt know we had a `testType` option

Reply:
changed